src/@next/EmptyState/EmptyState.tsx

```diff
@@ -24,6 +24,8 @@ export interface EmptyStateProps {
   fullWidth?: boolean;
   helpText?: string;
   imageName?: ImageName;
+  /** Renders a custom image */
```

Comment:
do we need to have a union here? we would erxpect either customImage | imageName to be present only